website/content/api-docs/query.mdx

```diff
@@ -478,6 +480,10 @@ $ curl \
 This endpoint executes an existing prepared query. If no query exists by the
 given ID, an error is returned.
 
+The HTTP response includes the `X-Consul-Results-Filtered-By-ACLs: true` header
```

Comment:
Another exception: this endpoint doesn't filter the entire response, just one of the fields in the response (`Nodes`).